libs/components/src/lib/role-selection-form/role-selection-form.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { StyledButton } from "@freelance/components";
+import { Form, RadioGroup } from "./role-selection-form.styled";
+
+export function RoleSelectionForm() {
+	const { t } = useTranslation();
+	const navigate = useNavigate();
+	const [radioOption, setRadioOption] = useState("");
```

Comment:
can we set null?
and you forget to add type

Reply:
fixed type

Reply:
fixed type

Reply:
fixed set null